appdev/guides/writeable-dirs.rst

```diff
@@ -0,0 +1,90 @@
+Writable directories
+====================
+
+App confinement is part of the Ubuntu Touch security concept. Data can be
+exchanged between apps only according to the AppArmor policies. This being said,
```

Comment:
We might want to add a link to the ContentHub guides to let people know where to learn more about exchanging data.

Reply:
Done. I hope, I made the link correctly.